package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fuse-v1",
-  "version": "0.1.0",
```

Comment:
I would prefer to leave this 0.1.0 seeing as it is a moving target. Any patches would then be `0.1.1`, `0.1.2` etc..